config/authors/aqsakausar30.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Aqsa Kausar",
   "bio": "Tuning my hyper-parameters everyday to dive deeper into the foray of deep learning.",
-  "photoURL": "https://miro.medium.com/fit/c/262/262/1*R53Abhu-jvhyQ2dANPTcBA.jpeg",
+  "photoURL": "Aqsa is a curious Machine Learning Engineer who is passionate about technology, learning new things and sharing ideas with others. As an ML Engineer she has hands-on experience with a diverse set of projects ranging from footfall forecasting to cataract detection. She has also worked on projects related to augmented reality, object detection and recommender systems.",
```

Comment:
This changes has to be in Bio not Photo URL.

Reply:
Ugh! How did that even happen :0
Fixed it.